packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.test.js

```diff
@@ -8,17 +8,33 @@ const props = {
 };
 
 test('controlled', () => {
-  const view = shallow(<Checkbox checked />);
+  const view = shallow(<Checkbox checked id="check" />);
   expect(view).toMatchSnapshot();
 });
 
 test('uncontrolled', () => {
-  const view = shallow(<Checkbox />);
+  const view = shallow(<Checkbox id="check" />);
   expect(view).toMatchSnapshot();
 });
 
 test('isDisabled', () => {
-  const view = shallow(<Checkbox isDisabled />);
+  const view = shallow(<Checkbox id="check" isDisabled />);
+  expect(view).toMatchSnapshot();
+});
+
+test('label is string', () => {
+  const view = shallow(<Checkbox label="Label" id="check" checked />);
+  expect(view).toMatchSnapshot();
+});
+
+test('label is function', () => {
```

Comment:
can you add test for making sure a custom class can be passed in, e.g. `class="test-123"` and one for passing a html attribute, e.g. `aria-labelledby="labelId"